src/sentry/static/sentry/app/components/confirm.tsx

```diff
@@ -40,7 +40,7 @@ type Props = {
   /**
    * Button priority
    */
-  priority: Button['props']['priority'];
+  priority: React.ComponentProps<typeof Button>['priority'];
```

Comment:
Should we be using this utility type instead of indexing by `props`?

Reply:
In the case of `<Button>`, because we now forward refs, the component type becomes something that does not have a 'props' index (see image). Using `React.ComponentProps` will help us get the props of a component.

![image](https://user-images.githubusercontent.com/79684/72647750-1a417e00-392e-11ea-9341-a573acb65b89.png)